spaceship/lib/spaceship/connect_api/api_client.rb

```diff
@@ -170,9 +170,16 @@ def with_asc_retry(tries = 5, &_block)
 
         return response
       rescue UnauthorizedAccessError => error
-        # Catch unathorized access and re-raising
-        # There is no need to try again
-        raise error
+        tries -= 1
+        puts(error) if Spaceship::Globals.verbose?
```

Comment:
Can we change this to a UI.warning instead of puts?

Reply:
Hey @rogerluan, thanks for the feedback.

When replacing `puts` with `UI.` in these locations, the linter complains with the following message:
```
💥 ensure_no_debug_code
[18:40:38]: Found debug code ' UI\.': 
```

It's also worth pointing out that every single instance of `if Spaceship::Globals.verbose?` is preceded by `puts(...)`.

How would you recommend to proceed here?

Reply:
Ah okay then. This is because we're not supposed to have `UI.…` code within spaceship lib. :) all good. Thanks for checking!